### PR TITLE
WIP: Update geth ipfs-ethdb and refactor.

### PR DESCRIPTION
**Pull request by arijitAD**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -21,3 +20,8 @@ require (
 	github.com/spf13/viper v1.7.0
 	github.com/vulcanize/ipfs-ethdb v0.0.5
 )
+
+replace (
+	github.com/ethereum/go-ethereum v1.10.11 => github.com/Vulcanize/go-ethereum v0.0.0-20211125055606-cd7c58e7f9a2
+	github.com/vulcanize/ipfs-ethdb v0.0.5 => github.com/Vulcanize/ipfs-ethdb v0.0.0-20211125060829-0aa16344859a
```
Comment (i-norden) [inline go.mod:26]: It doesn't seem to matter but it looks odd for Vulcanize to be capitalized in replace paths

File: go.mod [hunk 2]
```diff
@@ -3,15 +3,14 @@ module github.com/vulcanize/eth-ipfs-state-validator
 go 1.13
```
Comment (i-norden) [inline go.mod:3]: Let's bump to 1.5, since that's where the rest of the stack is currently

**Review by i-norden — APPROVED** [review]
LGMT, just a couple nit-picks.

Is there an issue associated with this (idt there is)?

**Comment by i-norden** [comment]
Actually I am going to echo what was said here: https://github.com/vulcanize/ipfs-ethdb/pull/23#issuecomment-1002724022

I don't think we want to add this additional dependency complexity for the hypothetical assumption that we want to use the pgx driver. And if we do want to use the pgx driver I think it is still cleaner to avoid depending on the sql.Database interface in our statediffing geth and instead just implement the driver usage directly here.

**Comment by ashwinphatak** [comment]
Closing as outdated.

**Pull request closed without merging**